source/_integrations/modbus.markdown

```diff
@@ -298,7 +298,7 @@ modbus:
         address: 100
         scan_interval: 20
         slave: 1
-      - name: "binary_ensor2"
+      - name: "binary_sensor2"
```

Comment:
What is wrong with the name? it is a perfect legal name for a sensor. I would not bother changing it, we have another plan of changing all the names to not include the entity type (like mea1 etc.) so nobody thinks that is a demand.

Reply:
It doesn't change anything functional, but it does look unprofessional to have typos. I'll therefore going to merge the change.